src/components/pages/Stripe/StripeCheckoutButton.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import StripeCheckout from 'react-stripe-checkout';
+//import axiosWithAuth from '../../../utils/axiosWithAuth';
+import axios from 'axios';
+
+const StripeCheckoutButton = ({ price }) => {
+  const priceForStripe = price * 100;
+  const publishablekKey =
```

Comment:
In the future, we'll want to abstract away our Stakeholder's actual Stripe key behind our `.env`s, but this should be okay for now.